messaging/src/main/java/org/frankframework/extensions/kafka/AbstractKafkaFacade.java

```diff
@@ -57,4 +63,26 @@ public void configure() throws ConfigurationException {
 		properties.setProperty(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getCanonicalName());
 		properties.setProperty(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getCanonicalName());
 	}
+
+	/**
+	 * There's no good way to check if the Kafka cluster is available, besides using the AdminClient which we have on board anyway.
+	 */
+	protected void checkConnection() {
+		Properties adminProperties = new Properties();
+		adminProperties.putAll(properties);
+
+		try (AdminClient adminClient = AdminClient.create(adminProperties)) {
+			DescribeClusterOptions describeClusterOptions = new DescribeClusterOptions();
+			describeClusterOptions.timeoutMs(5000);
+
+			DescribeClusterResult clusterResult = adminClient.describeCluster(describeClusterOptions);
+			KafkaFuture<String> clusterIdFuture = clusterResult.clusterId();
+			clusterIdFuture.get();
+		} catch (ExecutionException e) {
+			throw new LifecycleException("Didn't get a response from Kafka while connecting for Listening.", e);
```

Comment:
Wilde je niet nog checken of de `cause` een Timeout - exception was ipv een ander soort exception, zoals AuthorizationException?